odoo/models.py

```diff
@@ -2477,8 +2477,6 @@ def _read_group_prepare(self, orderby, aggregated_fields, annotated_groupbys, qu
                 orderby_terms.append(' '.join(order_split))
             elif order_field not in self._fields:
                 raise ValueError("Invalid field %r on model %r" % (order_field, self._name))
-            elif order_field == 'sequence':
-                pass
```

Comment:
So isn't read_group going to warn *again* in the cases where the ORM ignores its grouping requests?